### PR TITLE
support batch for mask head and pooler

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

support batch for mask head and pooler

## How was this patch tested?

unit tests

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/MaskHeadSpec.scala [hunk 1]
```diff
@@ -334,6 +334,332 @@ class MaskHeadSpec extends FlatSpec with Matchers {
     })
   }
 
+  "MaskHead with batch size > 1" should "be ok" in {
+    val inChannels: Int = 6
+    val resolution: Int = 14
+    val scales: Array[Float] = Array[Float](0.25f, 0.125f)
+    val samplingRratio: Float = 2.0f
```
Comment (benjamim93) [inline spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/MaskHeadSpec.scala]: Maybe name it to samplingRatio?

Comment (zhangxiaoli73) [reply spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/MaskHeadSpec.scala]: ok, changed

**Comment by benjamim93** [comment]
LGTM

**Comment by zhangxiaoli73** [comment]
jenkins pass： http://mlt-bddl01.sh.intel.com:8080/view/PR/job/BigDL-PRVN-lite/950/

**Pull request merged**